index.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  Modal: require("./src/Modal/Modal")
+}
```

Comment:
@jacobsimon would webpack be smart enough to not require `Modal` if I had `import {SomethingOtherThanModal} = from 'clever-components'`?